src/components/text-editor/text-editor.spec.tsx

```diff
@@ -921,27 +1011,32 @@ describe("TextEditor", () => {
   });
 
   describe("custom row prop type", () => {
-    describe("when value is number less than 2", () => {
-      it("throws an error", () => {
-        TextEditor.displayName = "EditorWithRowsLessThan2";
-        const errorMessage =
-          "Warning: Failed prop type: Prop `rows` must be a number value greater than 2 to override the min-height of the `EditorWithRowsLessThan2`";
+    let consoleSpy: jest.SpyInstance<
+      void,
+      [message?: unknown, ...optionalParams: unknown[]]
+    >;
+    beforeEach(() => {
+      consoleSpy = jest
+        .spyOn(global.console, "error")
+        .mockImplementation(() => undefined);
+    });
 
-        const assert = expectError(errorMessage);
-        render({ rows: 1 }, shallow);
-        assert();
-      });
+    afterEach(() => {
+      consoleSpy.mockRestore();
     });
 
-    describe("when value is not a number", () => {
-      it("throws an error", () => {
-        TextEditor.displayName = "EditorWithRowsAsString";
-        const errorMessage =
-          "Warning: Failed prop type: Prop `rows` must be a number value greater than 2 to override the min-height of the `EditorWithRowsAsString`";
+    afterAll(() => {
+      consoleSpy.mockClear();
+    });
 
-        const assert = expectError(errorMessage);
-        render({ rows: "foo" }, shallow);
```

Comment:
This test was removed because it failed the type check for `rows` as it can only be a number or undefined. So if someone were to do this in their project, type checking would fail anyway.

Reply:
Only in a TS project